tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -611,7 +611,7 @@ class TargetedSuite extends DatabaseSuite(SemanticdbMode.Slim) {
        |  [38..39): * => _star_.
        |[17..17): *.apply[Int]
        |  [0..1): * => _star_.
-       |  [2..7): apply => scala.collection.immutable.List.apply(A*).
+       |  [2..7): apply => scala.collection.immutable.List.apply(?).
```

Comment:
This is a good example where `?` hurts readability

Reply:
See above.